code/game/objects/effects/contraband.dm

```diff
@@ -53,6 +53,7 @@
 	var/ruined = FALSE
 	var/random_basetype
 	var/never_random = FALSE // used for the 'random' subclasses.
+	buildable_sign = FALSE //Cannot be unwrenched from a wall.
```

Comment:
Put the redefined vars over the newly defined ones, please, for proper organization.

Reply:
Alright.